### DISCUSSION TITLE
why my ohlcv dataframe interval is 15min, but in ploting, the candle bar interval is 2h?

### DISCUSSION THREAD

**Discussion by peilinnnnnn** [kernc/backtesting.py#997] [Q&A] [answered]
<img width="1004" alt="image" src="https://github.com/kernc/backtesting.py/assets/77475610/b4feac09-9607-4007-a862-4391e44632c3">
As the screenshot shows, my ohlcv dataframe interval is 15min, but in ploting, the candle bar interval is 2h. Can I know why, and how to make the candle bar interval is same as ohlcv dataframe interval?

**Comment by eervin123** [comment, marked as answer]
It down samples the plot if they're over I think 10,000 candles. However, if you want to see the fall resolution then pass resample=False to your plotting function.

Bt.plot(resample=False) should work.

**Reply by peilinnnnnn** [reply]
I see, it works, thanks a lot!
